src/components/List/List.jsx

```diff
@@ -15,7 +15,15 @@ const List = () => {
     <div>
       {error && <strong>Error: {JSON.stringify(error)}</strong>}
       {loading && <span>Collection: Loading...</span>}
-      {value && (
+
+      {value && value.empty && (
```

Comment:
Nit: we can use optional chaining rather than the `&&`

Reply:
Thanks, it has been solved. 👍🏻